src/noit_message_decoder.c

```diff
@@ -413,9 +414,9 @@ noit_metric_tags_count(const char *str, size_t strlen) {
   return rval;
 }
 
-const char *
-noit_metric_tags_parse_one(const char *tagnm, size_t tagnmlen,
-                           noit_metric_tag_t *output, mtev_boolean *toolong) {
+const char *parse_metric_tag(const char *const tagnm, const size_t tagnmlen,
+                             noit_metric_tag_t *output, mtev_boolean *toolong,
+                             const int32_t max_tag_length) {
```

Comment:
```suggestion
                             const size_t max_tag_length) {
```

Reply:
should 
```
static const int32_t NOIT_IMPLICIT_TAG_MAX_PAIR_LEN =
    MAX_METRIC_TAGGED_NAME + sizeof("__name:") - 1;
```
use `size_t` instead, as well? it is being passed to this parameter and is intended to always be used as a size.

Reply:
yes, ``sizeof`` returns a ``size_t``